File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/PromtionCriteriaClientModel.ts
export interface PromtionCriteriaClientModel {
    clientCriteriaId: number;
    promotionId: number | null;
    clientAttributeId: number | null;
    clientAttributeCode: string | null;

    valueFrom: string;

    excluded: boolean | null;
    isActive: boolean | null;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;
}

File: SofiForce.Models/Models/EntityModels/PromtionCriteriaClientModel.cs
using System;

namespace Models
{
    public class PromtionCriteriaClientModel
    {
        public int ClientCriteriaId { get; set; }
        public int? PromotionId { get; set; }
        public int? ClientAttributeId { get; set; }
        public string ClientAttributeCode { get; set; }

        
        public string ValueFrom { get; set; }
        public bool? Excluded { get; set; }
        public bool? IsActive { get; set; }
        public int? CBy { get; set; }
        public DateTime? CDate { get; set; }
        public int? EBy { get; set; }
        public DateTime? EDate { get; set; }
    }
}
